docs/contributing/maintainer_guide.rst

```diff
@@ -0,0 +1,75 @@
+Maintainer Guide
+================
+
+This is a guide for buildtest maintainers
+
+
+Incoming Pull Request
+------------------------
+
+These are just a few points to consider when dealing with incoming pull requests
+
+1. Any incoming Pull Request should be assigned to one or more maintainers for review.
+
+2. Upon approval, the PR should be **Squash and Merge**. If it's important to preserve a few commits during PR then **Rebase and Merge** is acceptable.
```

Comment:
I think there is a setting for this under the repo settings, I'll check after this final review.